src/app/shared/model/audio-version.model.ts

```diff
@@ -127,7 +127,7 @@ export class AudioVersionModel extends BaseModel {
   }
 
   changed(field?: string | string[], includeRelations = true): boolean {
-    if (this.isNew && this.files.length === 0) {
+    if (this.isNew && this.files.length === 0 && this.explicit === undefined) {
```

Comment:
On the question of what the default should be, do we support not including the tag `<itunes:explicit>` if explicit is not set or is empty string? Or do we always include the tag?